bundles/io.openliberty.tools.eclipse.lsp4e/plugin.xml

```diff
@@ -105,9 +104,10 @@
             editorId="org.eclipse.ui.genericeditor.GenericEditor">
       </editorContentTypeBinding>
    </extension>   
-   
+
    <extension
```

Comment:
So we have only one extension of the EP.. with three <server>(s) in it now.   Assume this is something we realized in early testing, and followed some other pattern?

Reply:
yes - i realized this after looking closer at the quarkus plugin which also has several LS incorporated into it under one LS EP. I decided to follow suit for ours as well as a best practice.